h/templates/groups/share.html.jinja2

```diff
@@ -32,17 +32,27 @@
         </ul>
     </div>
     {% endif %}
+    {% if member_usernames %}
+    <div class="group-members-list">
+      <ul class="group-members-list__list">
+        <p class="group-members-list__heading">Group Members:</p>
+        {% for member_username in member_usernames %}
+          <li>
+            <a href="{{ request.route_url('stream') }}?q=group:{{ group.pubid }} username: {{ member_username }}"
```

Comment:
Oh, and it should be `user:` (instead of `username:`), and without a space between the key and the value.